### PR TITLE
Added integration tests travis script.

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@ env:
...
   - secure: g+rMdXQTBwVb5S58Mj2uKbhwLE1N+Rc/3qGIEzfLY8V5Jj2beak8uNH+m/7dmRPBYLaL6QP5ndt6cY47w3lcXWGa00xeve12BHk9i5dvzx4hDZ18PoWGA5iB/U1Aj4I8USrASliyb2dqHtQg0V2nixgHuNdlJiwDgRNROHAdsQg=
 install: /bin/true
-script: ./ci/script/build.sh
-after_script:
-- ./ci/script/deploy.sh
+script:
+  - export BRANCH_NAME=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+#  - ./ci/script/build.sh
+  - ./ci/script/run_integrated_tests.sh
+after_success:
```
Comment (Pacane) [inline .travis.yml]: @kaendfinger Is that OK with you?

File: ci/script/run_integrated_tests.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+wget https://storage.googleapis.com/dart-archive/channels/stable/release/latest/sdk/dartsdk-linux-x64-release.zip
+unzip dartsdk-linux-x64-release.zip
+CURRENT_DIR="$(pwd)"
+
+git clone https://github.com/IOT-DSA/dslink-dart-test -b repacking-tested-links
```
Comment (Pacane) [inline ci/script/run_integrated_tests.sh]: The `-b` needs to be removed when https://github.com/IOT-DSA/dslink-dart-test/pull/10 gets merged. So don't merge this PR yet.

File: .travis.yml (hunk 1 above)
Comment (kaendfinger) [inline .travis.yml]: Disabling build.sh will prevent a lot of stuff from running, including deployments. Can you maybe merge the integrated test script into the build script?

Comment (Pacane) [inline .travis.yml]: I mean, I can still run both. The thing is, right now, the test project builds the link and the SDK, so I just wanted to save time and prevent it to build the link twice. I guess I can set it back, and add to a next PR a mechanism to use the built stuff from the build.sh script, if that makes sense.

**Pull request merged**